main/src/library/File.flix

```diff
@@ -364,4 +364,152 @@ namespace File {
             }
         }
 
+    ///
+    /// Writes `data` to the given file `f`.
+    ///
+    /// Creates file `f` if it does not exist. Overwrites it if it exists.
+    ///
+    /// Returns `true` if the file `f` was created, and `false` if `f` was overwritten.
```

Comment:
Is this description better for explaining what the method(s) returns?

Reply:
Yes, looks good :)

But for the `append` functions, they append to the file instead of overwriting it, so they need a slightly different description.